pkg/cmd/types.go

```diff
@@ -62,6 +64,7 @@ func NewDefaultImageInspectorOptions() *ImageInspectorOptions {
 		PasswordFile:   "",
 		ScanType:       "",
 		ScanResultsDir: "",
+		CVEUrlPath:     "",
```

Comment:
Could you put the default redhat.com url here, and pass around a mandatory url instead of optional `CVEUrlAltPath `?

If you decide to keep `CVEUrl` const, it should be renamed to something like `DefaultCVEUrl`.